### PR TITLE
BR, TiCDC: update br, cdc's docs and FAQ

### PR DESCRIPTION
**Pull request by lichunzhu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update br, cdc's documents and FAQ.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lichunzhu** [comment]
It sames that CI can't open some external links but I can open them on my own PC. 🤔 
```
connectors-and-apis.md
	ERROR	http://gnade.sourceforge.net/
		Dialing to the given TCP address timed out
	ERROR	http://octave.sourceforge.net/database/index.html
		Dialing to the given TCP address timed out
sql-statements/sql-statement-explain.md
	ERROR	http://www.webgraphviz.com/
		Dialing to the given TCP address timed out
```

**Comment by TomShawn** [comment]
> It sames that CI can't open some external links but I can open them on my own PC. 🤔
> 
> ```
> connectors-and-apis.md
> 	ERROR	http://gnade.sourceforge.net/
> 		Dialing to the given TCP address timed out
> 	ERROR	http://octave.sourceforge.net/database/index.html
> 		Dialing to the given TCP address timed out
> sql-statements/sql-statement-explain.md
> 	ERROR	http://www.webgraphviz.com/
> 		Dialing to the given TCP address timed out
> ```

We can ignore the external link check.

File: br/backup-and-restore-faq.md [hunk 1]
```diff
@@ -89,3 +89,11 @@ TiCDC 可以通过配置项中的 [`filter.rules`](https://github.com/pingcap/ti
 ## 在 Kubernetes 环境中如何使用 BR 进行增量备份？
 
 可以使用 kubectl 执行 `kubectl -n ${namespace} get bk ${name}` 以获得上次 BR 备份 `commitTs` 字段，该字段的内容可作为 `--lastbackupts` 使用。
+
+## BR 恢复存档后是否需要 analyze 表，以及如何 analyze？
```
Comment (kennytm) [inline br/backup-and-restore-faq.md:93]:
```suggestion
## BR 恢复存档后是否需要 ANALYZE 表？
```

(the entry did not answer "如何 analyze" so let's remove this part from the question)

File: br/backup-and-restore-use-cases.md [hunk 2]
```diff
@@ -179,30 +179,40 @@ bin/br backup table \
 
 #### 结果解读
 
-使用 BR 前已设置日志的存放路径。从路径下存放的日志中可以获取此次备份的相关统计信息。在日志中搜关键字 "summary"，可以看到以下信息：
+BR 会在备份结束时输出备份 summary 到控制台。
```
Comment (kennytm) [inline br/backup-and-restore-use-cases.md:182]:
```suggestion
BR 会在备份结束时输出备份总结到控制台。
```

File: br/backup-and-restore-use-cases.md [hunk 3]
```diff
@@ -179,30 +179,40 @@ bin/br backup table \
...
+* 程序运行总耗时：`total take(real time): 2m33.213625285s`
+* 备份数据大小：`total size(MB): 5997.49`
+* 备份吞吐：`avg speed(MB/s): 67.1`
+* 校验耗时：`["backup checksum"=1m3.588888373s]`
+* 计算各表 checksum, kvs, bytes 信息总和的耗时：`["backup fast checksum"=358µs]`
+* 备份 region 总数：`["backup total regions"=2]`
+* 备份存档压缩后在磁盘中的实际大小：`[Size=826624755]`
+* 备份存档的快照时间戳：`[BackupTS=422550536188329985]`
+
+通过以上数据可以计算得到单个 TiKV 实例的吞吐为：`avg speed(MB/s)`/`tikv_count` = `67.1`。
```
Comment (kennytm) [inline br/backup-and-restore-use-cases.md:215]: the tikv_count in the old document is 4. now it is 1 which makes the calculation look pointless. please consider keeping the original numbers.

File: ticdc/troubleshoot-ticdc.md [hunk 4]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
     ```shell
     cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
     ```
+
+## 使用 TiCDC 创建 changefeed 时报错 `[tikv:9006]GC life time is shorter than transaction duration, transaction starts at xx, GC safe point is yy`
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:348]: @amyangfei PTAL, thanks!

File: ticdc/troubleshoot-ticdc.md [hunk 5]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+解决方案：需要使用 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 指令查询当前的 gc_safe_point 与 service_gc_safe_points。如果 gc_safe_point 小于 CDC 的 changefeed，则用户直接在 create changefeed 指令后加上 `--disable-gc-check` 参数创建 changefeed 即可。
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:350]:
```suggestion
解决方案：需要使用 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 指令查询当前的 gc_safe_point 与 service_gc_safe_points。如果 gc_safe_point 小于 CDC changefeed 的 `start-ts`，则用户直接在 create changefeed 指令后加上 `--disable-gc-check` 参数创建 changefeed 即可。
```

File: ticdc/troubleshoot-ticdc.md [hunk 6]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+如果上述指令结果中没有 gc_worker service_id 项目：
+
+1. PD 的版本 <= v4.0.8，该问题对应于已知问题 [pd#3128](https://github.com/tikv/pd/issues/3128)
+2. PD 是由 v4.0.8 及更低版本滚动升级到新版，该问题对应于已知问题 [pd#3366](https://github.com/tikv/pd/issues/3366)
+
+如果不满足上述情况，请将上述指令执行结果反应到 asktug 论坛或对应的支持 DBA。
+
+## 使用 TiCDC 创建 changefeed enable-old-value 设置为 true 后 cdc 执行到下游的语句仍然为 REPLACE INTO 而非 UPDATE
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:359]:
```suggestion
## 使用 TiCDC 创建 changefeed enable-old-value 设置为 true 后 cdc 执行到下游的语句仍然为 REPLACE INTO 而非 INSERT/UPDATE
```

**Comment by amyangfei** [comment]
lgtm for cdc part

**Comment by lichunzhu** [comment]
@kennytm @overvenus PTAL again

File: br/backup-and-restore-faq.md [hunk 7]
```diff
@@ -89,3 +89,11 @@ TiCDC 可以通过配置项中的 [`filter.rules`](https://github.com/pingcap/ti
 ## 在 Kubernetes 环境中如何使用 BR 进行增量备份？
 
 可以使用 kubectl 执行 `kubectl -n ${namespace} get bk ${name}` 以获得上次 BR 备份 `commitTs` 字段，该字段的内容可作为 `--lastbackupts` 使用。
+
+## BR 恢复存档后是否需要 ANALYZE 表？
+
+BR v4.0.10 及之前的版本（v4.0.9 除外）均不会备份统计信息。所以在恢复后需要手动执行 ANALYZE TABLE 或等待 TiDB 自动进行 ANALYZE。
```
Comment (kennytm) [inline br/backup-and-restore-faq.md:95]:
```suggestion
BR 不会备份统计信息（v4.0.9 除外）。所以在恢复后需要手动执行 ANALYZE TABLE 或等待 TiDB 自动进行 ANALYZE。
```

i think we shouldn't mention v4.0.10 here. we can include the version _after_ the backup-stats feature is released.

**Review by kennytm — COMMENTED** [review]
rest LGTM

**Comment by kennytm** [comment]
LGTM

File: ticdc/troubleshoot-ticdc.md [hunk 8]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
     ```shell
     cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
     ```
+
+## 使用 TiCDC 创建 changefeed 时报错 `[tikv:9006]GC life time is shorter than transaction duration, transaction starts at xx, GC safe point is yy`
+
+解决方案：需要使用 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 指令查询当前的 gc_safe_point 与 service_gc_safe_points。如果 gc_safe_point 小于 CDC changefeed 的 `start-ts`，则用户直接在 create changefeed 指令后加上 `--disable-gc-check` 参数创建 changefeed 即可。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:350]:
```suggestion
解决方案：需要执行 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 命令查询当前的 GC safepoint 与 service GC safepoint。如果 GC safepoint 小于 TiCDC changefeed 同步任务的开始时间戳 `start-ts`，则用户可以直接在 `cdc cli create changefeed` 命令后加上 `--disable-gc-check` 参数创建 changefeed。
```

File: ticdc/troubleshoot-ticdc.md [hunk 9]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+如果上述指令结果中没有 gc_worker service_id 项目：
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:352]:
```suggestion
如果上述命令结果中没有 `gc_worker service_id`：
```

File: ticdc/troubleshoot-ticdc.md [hunk 10]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+1. PD 的版本 <= v4.0.8，该问题对应于已知问题 [pd#3128](https://github.com/tikv/pd/issues/3128)
+2. PD 是由 v4.0.8 及更低版本滚动升级到新版，该问题对应于已知问题 [pd#3366](https://github.com/tikv/pd/issues/3366)
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:355]:
```suggestion
+ 如果 PD 的版本 <= v4.0.8，详见 [PD issue #3128](https://github.com/tikv/pd/issues/3128)
+ 如果 PD 是由 v4.0.8 或更低版本滚动升级到新版，详见 [PD issue #3366](https://github.com/tikv/pd/issues/3366)
```

File: ticdc/troubleshoot-ticdc.md [hunk 11]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+如果不满足上述情况，请将上述指令执行结果反应到 asktug 论坛或对应的支持 DBA。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:357]: 能否添加 AskTUG 的对应论坛链接？

File: ticdc/troubleshoot-ticdc.md [hunk 12]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+## 使用 TiCDC 创建 changefeed enable-old-value 设置为 true 后 cdc 执行到下游的语句仍然为 REPLACE INTO 而非 INSERT/UPDATE
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:359]:
```suggestion
## 使用 TiCDC 创建同步任务时将 `enable-old-value` 设置为 `true` 后 TiCDC 执行到下游的语句仍然为 `REPLACE INTO` 而非 `INSERT`/`UPDATE`
```

File: ticdc/troubleshoot-ticdc.md [hunk 13]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+需要在创建 changefeed 时显式指定 safe-mode 为 false 才会生成 UPDATE 语句，执行步骤为:
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:361]:
```suggestion
要解决该问题，创建 changefeed 时需要显式指定 `safe-mode` 为 `false` 才会生成 `UPDATE` 语句，执行步骤如下？
```

File: ticdc/troubleshoot-ticdc.md [hunk 14]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+```
+tiup ctl cdc changefeed pause -c simple-replication-task
+tiup ctl cdc changefeed update -c simple-replication-task --sink-uri "mysql://root:@127.0.0.1:3306/?safe-mode=false" --config ./changefeed.toml
+tiup ctl cdc changefeed resume -c simple-replication-task
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:366]: 可否将每条命令单独分开并加上描述？

File: ticdc/troubleshoot-ticdc.md [hunk 15]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+```
+
+值得注意的是目前修改 safe-mode 功能并没有对外开放，因为 TiCDC 还不提供 exactly once delivery 的保证；
+
+开启 safe-mode=false，因为有重复消息的存在，会出现同步中断的风险。因此，**不建议使用**。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:371]: 用户无法使用一个还没开放给 TA 的方法去解决问题，那这个问题的回答就没有意义了吧？

File: ticdc/troubleshoot-ticdc.md [hunk 16]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+## 使用 TiCDC 同步消息到 kafka 时 kafka 报错 Message was too large
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:373]:
```suggestion
## 使用 TiCDC 同步消息到 Kafka 时 Kafka 报错 `Message was too large`
```

File: ticdc/troubleshoot-ticdc.md [hunk 17]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+v4.0.8 与更低版本的 TiCDC，Sink URI 配置 kafka 中的 `max-message-bytes` 参数不能保证有效控制输出到 kafka 的消息大小，需要在 kafka server 配置中加入下述配置以提高接受消息的字节数限制。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:375]:
```suggestion
v4.0.8 或更低版本的 TiCDC，仅在 Sink URI 中为 Kafka 配置 `max-message-bytes` 参数不能有效控制输出到 Kafka 的消息大小，需要在 Kafka server 配置中加入如下配置以增加 Kafka 接收消息的字节数限制。
```

File: ticdc/troubleshoot-ticdc.md [hunk 18]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+```
+#broker能接收消息的最大字节数
+message.max.bytes=2147483648
+#broker可复制的消息的最大字节数
+replica.fetch.max.bytes=2147483648
+#消费者端的可读取的最大消息
+fetch.message.max.bytes=2147483648
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:383]:
```suggestion
# broker 能接收消息的最大字节数
message.max.bytes=2147483648
# broker 可复制的消息的最大字节数
replica.fetch.max.bytes=2147483648
# 消费者端的可读取的最大消息字节数
fetch.message.max.bytes=2147483648
```

File: br/backup-and-restore-faq.md [hunk 19]
```diff
@@ -89,3 +89,11 @@ TiCDC 可以通过配置项中的 [`filter.rules`](https://github.com/pingcap/ti
 ## 在 Kubernetes 环境中如何使用 BR 进行增量备份？
 
 可以使用 kubectl 执行 `kubectl -n ${namespace} get bk ${name}` 以获得上次 BR 备份 `commitTs` 字段，该字段的内容可作为 `--lastbackupts` 使用。
+
+## BR 恢复存档后是否需要 ANALYZE 表？
+
+BR 不会备份统计信息（v4.0.9 除外）。所以在恢复后需要手动执行 ANALYZE TABLE 或等待 TiDB 自动进行 ANALYZE。
```
[the remark this answers is not in the document]
Comment (lichunzhu) [reply br/backup-and-restore-faq.md:95]: 不会

File: ticdc/troubleshoot-ticdc.md (hunk 11 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:357]:
```suggestion
如果不满足上述情况，请将上述指令执行结果反应到 [AskTUG 论坛](https://asktug.com/tags/ticdc) 或对应的支持 DBA。
```

File: ticdc/troubleshoot-ticdc.md (hunk 13 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:361]:
```suggestion
要解决该问题，创建 changefeed 时需要显式指定 `safe-mode` 为 `false` 才会生成 `UPDATE` 语句，执行步骤如下：
```

File: ticdc/troubleshoot-ticdc.md (hunk 14 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:366]: 感觉没啥必要，就是更新任务配置的操作，pause 任务，更新任务，resume 任务

File: ticdc/troubleshoot-ticdc.md (hunk 15 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:371]: 我觉得后面可以再更新这里

**Comment by overvenus** [comment]
LGTM

File: br/backup-and-restore-faq.md [hunk 20]
```diff
@@ -89,3 +89,11 @@ TiCDC 可以通过配置项中的 [`filter.rules`](https://github.com/pingcap/ti
 ## 在 Kubernetes 环境中如何使用 BR 进行增量备份？
 
 可以使用 kubectl 执行 `kubectl -n ${namespace} get bk ${name}` 以获得上次 BR 备份 `commitTs` 字段，该字段的内容可作为 `--lastbackupts` 使用。
+
+## BR 恢复存档后是否需要对表执行 `ANALYZE` 以更新 TiDB 在表和索引上留下的统计信息？
+
+BR 不会备份统计信息（v4.0.9 除外）。所以在恢复后需要手动执行 ANALYZE TABLE 或等待 TiDB 自动进行 ANALYZE。
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:95]:
```suggestion
BR 不会备份统计信息（v4.0.9 除外）。所以在恢复存档后需要手动执行 `ANALYZE TABLE` 或等待 TiDB 自动进行 `ANALYZE`。
```

File: ticdc/troubleshoot-ticdc.md [hunk 21]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
     cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
     ```
+
+## 使用 TiCDC 创建 changefeed 时报错 `[tikv:9006]GC life time is shorter than transaction duration, transaction starts at xx, GC safe point is yy`
+
+解决方案：需要执行 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 命令查询当前的 GC safepoint 与 service GC safepoint。如果 GC safepoint 小于 TiCDC changefeed 同步任务的开始时间戳 `start-ts`，则用户可以直接在 `cdc cli create changefeed` 命令后加上 `--disable-gc-check` 参数创建 changefeed。
+
+如果上述命令结果中没有 `gc_worker service_id`：
+
++ 如果 PD 的版本 <= v4.0.8，详见 [PD issue #3128](https://github.com/tikv/pd/issues/3128)
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:354]: 下面已经说了 PD 小于等于 4.0.8 的情况
```suggestion
+ 如果 PD 的版本 > v4.0.8，详见 [PD issue #3128](https://github.com/tikv/pd/issues/3128)
```

File: ticdc/troubleshoot-ticdc.md [hunk 22]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
     ```shell
     cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
     ```
+
+## 使用 TiCDC 创建 changefeed 时报错 `[tikv:9006]GC life time is shorter than transaction duration, transaction starts at xx, GC safe point is yy`
+
+解决方案：需要执行 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 命令查询当前的 GC safepoint 与 service GC safepoint。如果 GC safepoint 小于 TiCDC changefeed 同步任务的开始时间戳 `start-ts`，则用户可以直接在 `cdc cli create changefeed` 命令后加上 `--disable-gc-check` 参数创建 changefeed。
+
+如果上述命令结果中没有 `gc_worker service_id`：
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:352]: 是说 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 中没有 `gc_worker service_id` 吗？

File: ticdc/troubleshoot-ticdc.md [hunk 23]
```diff
@@ -344,3 +344,41 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
++ 如果 PD 的版本 <= v4.0.8，详见 [PD issue #3128](https://github.com/tikv/pd/issues/3128)
++ 如果 PD 是由 v4.0.8 或更低版本滚动升级到新版，详见 [PD issue #3366](https://github.com/tikv/pd/issues/3366)
+
+如果不满足上述情况，请将上述指令执行结果反应到 [AskTUG 论坛](https://asktug.com/tags/ticdc) 或对应的支持 DBA。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:357]: 不满足`上述`情况是上面提到的两个`如果`吗？

File: ticdc/troubleshoot-ticdc.md (hunk 15 above)
Comment (TomShawn) [reply ticdc/troubleshoot-ticdc.md:371]: 1. `创建 changefeed 时需要显式指定 `safe-mode` 为 `false` 才会生成 `UPDATE` 语句，执行步骤如下`  --> 2. `修改 safe-mode 功能并没有对外开放` --> 3. `不建议使用`

这个逻辑似乎是混乱的。既然 safe-mode 功 都没对外开放了，为什么还要有 1 呢？

“我觉得后面开放后可以再更新这里”   等开放后再告诉用户创建 changefeed 时需要显式指定 `safe-mode` 为 `false` 才会生成 `UPDATE` 语句吧？

File: ticdc/troubleshoot-ticdc.md (hunk 22 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:352]: 是的

File: ticdc/troubleshoot-ticdc.md (hunk 21 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:354]: 不一样，一个是本身 PD 版本 <= 4.0.8，一个是 PD 虽然是新版，但是是由 <= 4.0.8 的版本滚动升级上来的

File: ticdc/troubleshoot-ticdc.md (hunk 23 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:357]: 是的

File: ticdc/troubleshoot-ticdc.md (hunk 15 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:371]: 但是 `enable-old-value` 已经开放给了用户，用户困惑的是为什么设置 `enable-old-value` 仍然还是 REPLACE。这里没有开放 safe-mode 我觉得可以不讲解决方案，但要告诉用户原因。

File: ticdc/troubleshoot-ticdc.md (hunk 23 above)
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:357]:
```suggestion
+ 对于其他情况，请将上述命令执行结果反馈到 [AskTUG 论坛](https://asktug.com/tags/ticdc)。
```

File: ticdc/troubleshoot-ticdc.md (hunk 15 above)
Comment (TomShawn) [reply ticdc/troubleshoot-ticdc.md:371]: 可以的。如果 safe-mode 未暴露，那么从用户角度就是暂时无解决办法。请补充一下原因吧。

File: ticdc/troubleshoot-ticdc.md [hunk 24]
```diff
@@ -345,6 +345,35 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+解决方案：需要执行 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 命令查询当前的 GC safepoint 与 service GC safepoint。如果 GC safepoint 小于 TiCDC changefeed 同步任务的开始时间戳 `start-ts`，则用户可以直接在 `cdc cli create changefeed` 命令后加上 `--disable-gc-check` 参数创建 changefeed。
+
+如果 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 的结果中没有 `gc_worker service_id`：
+
++ 如果 PD 的版本 <= v4.0.8，详见 [PD issue #3128](https://github.com/tikv/pd/issues/3128)
++ 如果 PD 是由 v4.0.8 或更低版本滚动升级到新版，详见 [PD issue #3366](https://github.com/tikv/pd/issues/3366)
++ 对于其他情况，请将上述命令执行结果反馈到 [AskTUG 论坛](https://asktug.com/tags/ticdc)。
+
+## 使用 TiCDC 创建同步任务时将 `enable-old-value` 设置为 `true` 后 TiCDC 执行到下游的语句仍然为 `REPLACE INTO` 而非 `INSERT`/`UPDATE`
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:358]: 是“TiCDC 执行到下游的 `INSERT`/`UPDATE` 语句仍然是 `REPLACE INTO`”吗？

File: ticdc/troubleshoot-ticdc.md [hunk 25]
```diff
@@ -345,6 +345,35 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+因为 TiCDC 创建 changefeed 时会默认指定 `safe-mode` 为 `true`，从而生成 `REPLACE INTO` 的执行语句。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:360]: 是为什么语句生成 `REPLACE INTO` 呢？ 是为 `INSERT`/`UPDATE` 语句吗？

File: ticdc/troubleshoot-ticdc.md (hunk 24 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:358]: 上游的 `INSERT`/`UPDATE` 经 TiCDC 同步到下游的语句仍然为 `REPLACE INTO`，不知道这样说你能理解吗

File: ticdc/troubleshoot-ticdc.md (hunk 25 above)
Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:360]: 上游的 INSERT/UPDATE 经 TiCDC 同步到下游会变为 REPLACE INTO

File: ticdc/troubleshoot-ticdc.md (hunk 24 above)
Comment (TomShawn) [reply ticdc/troubleshoot-ticdc.md:358]:
```suggestion
## 使用 TiCDC 创建同步任务时将 `enable-old-value` 设置为 `true` 后，上游的 `INSERT`/`UPDATE` 语句经 TiCDC 同步到下游后语句为 `REPLACE INTO`
```

File: ticdc/troubleshoot-ticdc.md (hunk 25 above)
Comment (TomShawn) [reply ticdc/troubleshoot-ticdc.md:360]:
```suggestion
TiCDC 创建 changefeed 时会默认指定 `safe-mode` 为 `true`，从而为上游的 `INSERT`/`UPDATE` 语句生成 `REPLACE INTO` 的执行语句。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**